tests/adapter.test.ts

```diff
@@ -95,6 +95,155 @@ function generateSuite(triple: string) {
                 await ds.terminate();
             });
 
+            test('set breakpoint (absolute path)', async function () {
+                let ds = await DebugTestSession.start(adapterLog);
+                let bpLineSource = findMarker(debuggeeSource, '#BP1');
+                let setBreakpointAsyncSource = ds.setBreakpoint(debuggeeSource, bpLineSource);
+
+                let stopAsync = ds.waitForEvent('stopped');
+                await ds.launch({ name: 'set breakpoint (absolute path)', program: debuggee, cwd: path.dirname(debuggee) });
+
+                const breakPointResponse: dp.SetBreakpointsResponse = await setBreakpointAsyncSource;
+                assert.deepEqual(breakPointResponse.body.breakpoints, [
```

Comment:
btw, these asserts may be too verbose and could hide what is really being tested here... lmk if you prefer to just assert `source.path` (I initially did this because there were no tests checking the returned object from setBreakpoint, so thought it may be a good addition).